File: ServidorGaia/src/main/java/domain/model/Friend.java
package domain.model;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class Friend {
    private String username1;
    private String username2;
    private int value;
    private LocalDateTime requestDate;

    public Friend(String usuario1, String usuario2, int valor, LocalDateTime fecha_solicitud) {
        this.username1 = usuario1;
        this.username2 = usuario2;
        this.value = valor;
        this.requestDate = fecha_solicitud;
    }

    public Friend() {
    }
}

File: UIGaia/app/src/main/java/com/example/uigaia/domain/model/Friend.kt
package com.example.uigaia.domain.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import java.time.LocalDate

@Parcelize
data class Friend (
    val firstUser: String,
    val secondUser: String,
    val status: Int,
    val requestDate: LocalDate
        ) : Parcelable